example/src/main/kotlin/com/tobrun/data/compat/example/PersonData.kt

```diff
@@ -15,7 +15,7 @@ annotation class SampleAnnotation
 @DataCompat
 @SampleAnnotation
 private data class PersonData(
-    @Default("\"John\"")
+    @Default("\"John\" + Date(1580897313933L).toString()", importList = ["java.util.Date"])
```

Comment:
nit:
```suggestion
    @Default("\"John ${Date(1580897313933L)}\"", importList = ["java.util.Date"])
```

Reply:
using `$` is not allowed within "" - I did not want to mess up with escaping it and so on

Reply:
ah, okay